storage/innobase/handler/ha_innodb.cc

```diff
@@ -6939,7 +6952,10 @@ wsrep_store_key_val_for_row(
 				}
 				memcpy(buff, sorted, true_len);
 			} else {
-				memcpy(buff, src_start, true_len);
+				ut_ad(src_start || (src_start == nullptr && true_len == 0));
+				if (src_start) {
+					memcpy(buff, src_start, true_len);
+				}
 			}
```

Comment:
While here even a register-starved ISA like IA-32 might be able to hold all of `real_type`, `mysql_type` (which occur in the `if` condition), `src_start`, and `true_len` in registers (without any spilling to stack), I would still make this uniform with my suggestion to the other occurrences:
```c++
			} else if (true_len) {
				memcpy(buff, src_start, true_len);
			} else {
				ut_ad(!src_start);
			}
```